Automattic/wp-calypso
# Earn: Migrate Images to Webpack

#### Changes proposed in this Pull Request

This migrates the images of the Earn section to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There's three routes which need to be checked:

- `/devdocs/design/promo-section`
- `/devdocs/design/promo-card`
- `/earn/site`

In all cases, verify that the images are still there. Also confirm that there's no other usage of these images.

cc @pablinos, @davemart-in 

<img width="1117" alt="Screenshot 2019-10-17 at 17 04 27" src="https://user-images.githubusercontent.com/43215253/67026939-a7dbba80-f100-11e9-91f7-2e264c5d8e6e.png">
